Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al

```diff
@@ -253,6 +254,36 @@ codeunit 20352 "Connectivity App Definitions"
         RegisterApp(AppId, AppName, AppPublisher, AppDescription, AppProviderSupportURL, AppSourceURL, AppApprovedFor, AppWorksOn, "Connectivity Apps Category"::Banking);
     end;
 
+    local procedure RegisterSofteraBankfeed()
+    var
+        AppId: Text[250];
+        AppName: Text[1024];
+        AppPublisher: Text[250];
+        AppDescription: Text[2048];
+        AppProviderSupportURL: Text[250];
+        AppSourceURL: Text[250];
+        AppWorksOn: Text;
+        AppApprovedFor: Text;
+    begin
+        /***************************************************
+            Add app 'Bankfeed - Bank Statement Import & Reconciliation' to LT, DK, HU
+        ***************************************************/
+
+        AppId := '8051ee19-8246-489f-9f2c-4d89e1710f7e';
+        AppName := 'Bankfeed - Bank Statement Import & Reconciliation';
+        AppPublisher := 'Softera Baltic';
+        AppDescription := 'Bank statement import & reconciliation.';
+        AppProviderSupportURL := 'https://www.softera.lt/en/banking-automation-payment-management-bankfeed/';
```

Comment:
@lukasaus The link to supported banks should go directly to the place where the user can see which banks are supported. I suggest you change it to this one: https://www.softera.lt/en/supported-banks-softera-bankfeed/